modules/030-cloud-provider-aws/images/cloud-controller-manager/patches/1.23/README.md

```diff
@@ -0,0 +1,12 @@
+## Patches
+
+## 001-identify-instances-by-name.patch
+
+Find nodes in the cloud by the `Name` tag containing Node privateDNSName.
+
+## 002-non-type-lb.patch
+
```

Comment:
🤔

Reply:
> restarts of all control plane components
Agree. But it's really uncomfortable to have different behavior. Everywhere we have patches with -p1 and only here it's different. And on update and patch backport it's really painful

Reply:
Where is information about `002-non-type-lb.patch`?

Reply:
Added docs